app/controllers/SignOutController.scala

```diff
@@ -39,18 +37,20 @@ class SignOutController @Inject() (
   def signOutFromTimeout: Action[AnyContent] = Action { implicit request =>
     // N.B. the implicit request being passed into the page here may still have the auth
     // token in it so take care to ensure that the sign out link is not shown by mistake
-    Ok(views.timedOutPage()).withNewSession
+    val continueUrl = routes.SignOutController.signOutFromTimeout.absoluteURL()
```

Comment:
won't this result in an infinite redirect loop? You may need another controller endpoint which just serves the `views.timedOutPage` content to redirect to here

Reply:
oh yeah, but it will be spectacular
..will fix